coldfront/plugins/qumulo/forms/UpdateAllocationForm.py

```diff
@@ -18,3 +20,15 @@ def __init__(self, *args, **kwargs):
             required=False,
         )
         self.fields["prepaid_expiration"].disabled = True
+        self.fields["rw_users"].required = self._rw_user_required()
```

Comment:
If we just pass status_name instead of the allocation Id, we save even more Queries
```suggestion
        status_name = kwargs.pop("status_name", None)
        self.fields["rw_users"].required = status_name != "Ready for deletion"
```